setup.py

```diff
@@ -13,7 +13,7 @@
       packages=[],
       install_requires=[
         'lxml >= 4.3.0',
-        'pyVmomi >= 6.7',
+        'pyVmomi <=7.0.3, >=6.7',
```

Comment:
Is there something compatiblity issue with Pyvmomi 8? It just shipped.

https://github.com/vmware/pyvmomi/releases

Reply:
Yes, https://github.com/vmware/pyvmomi/issues/993

Reply:
Got it. We should se how to enable somehow 8.0.0.1.patch1 whenever it comes. I se a fix is in place already but no new release.